src/lib/toolbar/toolbar.ts

```diff
@@ -39,15 +42,43 @@ export const _MdToolbarMixinBase = mixinColor(MdToolbarBase);
   inputs: ['color'],
   host: {
     'class': 'mat-toolbar',
-    'role': 'toolbar'
+    'role': 'toolbar',
```

Comment:
This `role` should be removed; I never noticed it being added. It's not appropriate for most uses of the md-toolbar.
(see the docs)

Reply:
Good point